docs/Project.toml

```diff
@@ -2,4 +2,4 @@
 Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
 
 [compat]
-Documenter = "~0.20"
+Documenter = "0.23"
```

Comment:
Documenter.jl [recommends](https://juliadocs.github.io/Documenter.jl/stable/man/hosting/#.travis.yml-Configuration-1) using `~`, so I think we should do the same. Cc: @mortenpi

Reply:
it changed: https://github.com/JuliaDocs/Documenter.jl/pull/1105